src/frontend/src/handler/create_sink.rs

```diff
@@ -558,6 +558,7 @@ pub(crate) async fn reparse_table_for_sink(
         None,
         include_column_options,
         engine,
+        SqlColumnStrategy::Follow,
```

Comment:
Strictly follow the columns defined in SQL and avoid refreshing. Creating a new sink into a table should not alter the table's schema.